prow/jobs.yaml

```diff
@@ -178,3 +178,13 @@ kubernetes/test-infra:
   spec:
     containers:
     - image: gcr.io/k8s-testimages/test-infra-go-test:0.4
+
+- name: pull-test-infra-py-test
+  always_run: true
+  skip_report: true
+  context: py test
```

Comment:
should we call this "prow py test"? or rename the other one from "prow go test" to just "go test?"